client/landing/stepper/declarative-flow/internals/steps-repository/choose-a-domain/index.tsx

```diff
@@ -122,6 +128,12 @@ const ChooseADomain: Step = function ChooseADomain( { navigation, flow } ) {
 					isCartPendingUpdate={ isCartPendingUpdate }
 					isCartPendingUpdateDomain={ isCartPendingUpdateDomain }
 				/>
+				<div>
+					<Button variant="tertiary" onClick={ () => handleGoBack( goBack ) }>
+						<Gridicon icon="chevron-left" />
+						{ __( 'Back' ) }
+					</Button>
+				</div>
```

Comment:
This feels like it's the wrong way to solve this -- these buttons should be within the main Stepper components, and it looks like that is controlled via the `shouldHideNavButtons` prop on `StepContainer`:
https://github.com/Automattic/wp-calypso/blob/f9cbb14fe86a3df8b4f67189b06b193db8ce263c/client/landing/stepper/declarative-flow/internals/steps-repository/choose-a-domain/index.tsx#L240

For that line, `isBlogOnboardingFlow()` is true for the `start-writing` and `design-first` flows.

Reply:
Alrighty, I can enable it on the step navigation and then use CSS to nudge it over so there's no overlap with the logo. Thank you!